cmake/Dependencies.cmake

```diff
@@ -246,6 +246,7 @@ if(NOT MKL_FOUND)
   find_path(POCKETFFT_INCLUDE_DIR NAMES pocketfft_hdronly.h
             PATHS /usr/local/include
             PATHS $ENV{POCKETFFT_HOME}
+            PATH "${PROJECT_SOURCE_DIR}/third_party/pocketfft"
```

Comment:
maybe I am unqualified, but shouldn't this be `PATHS`? 
Also shouldn't there be only one `PATHS` and you apply each option space separated?

```cmake
find_path(POCKETFFT_INCLUDE_DIR NAMES pocketfft_hdronly.h PATHS 
    /usr/local/include 
    $ENV{POCKETFFT_HOME} 
    "${PROJECT_SOURCE_DIR}/third_party/pocketfft")
```

Reply:
And this might also the problems I had when building described here: https://github.com/pytorch/pytorch/pull/60976#issuecomment-893281689

Reply:
Thank you for spotting, updated! But I think there are few more things that needs to be fixed in mobile build.